NewForce-Cohort-6/tabloidfullstack-mashed-potatoes
# 10 create a category

# Description
[Trello Ticket #10 (https://trello.com/c/YoI0xvJm)](https://trello.com/c/wJTpQxV2)

Added functionality to add a new category to the category list. The new category can be saved by clicking the save button which will add the new category to the list and the user will then be directed to the category list. The list is sorted in A-Z order.

# Testing Instructions
1. `git add` and `git commit` whatever you're working on on your branch
2. `git checkout main`
3. `git fetch --a`
4. `git checkout 10CreateACategory`
5. If you don't see the most recent version of my files, `git pull origin 10CreateACategory'
6. If you haven't created the Tabloid database, please do so.
Use this data to fill out tables and columns: [SQL Tables](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/01_Tabloid_Create_DB.sql)
Use this to seed the database : [SQL SEED DATA](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/02_Tabloid_Seed_Data.sql)
7. Click the execute with debugger solid green button to launch the app
8. Log into the website using “leo@gmail.com” if greeted with a login page
9. From the home page, click the “Category Management” option in the navbar
10. The url should change to `/categories` and a list of categories, a create category heading and new category button should appear.
11. Click the new category button which will take you to the create category form at the url of '/createCategory'
12. Enter a new category in the text box and click Save.
13. The user will then be redirected back to the category list that will include the new category in A-Z order.



# Checklist:
